CONTRIBUTING_WITH_GIT2R.Rmd

```diff
@@ -0,0 +1,551 @@
+```
+
+# Introduction
+
+## Background/Motivation
+
+The **popgenInfo** project was designed to be an educational tool that can move
+at the same pace as method development by allowing people skilled in population
+genetics analysis to submit workflows that demonstrate the strengths of any
+given analysis in R. We use [git](https://git-scm.com/) and
+[GitHub](https://github.com) to accept contributions via pull requests using
+[GitHub flow](https://guides.github.com/introduction/flow/).
+
+> **What is git?**
+> 
+> Git is a *version control* program. It allows you to keep track of all changes
+that happen when you are writing, analyzing data, or coding. You can think of it
+as a sort of electronic lab notebook. Just like you would record any weights or
+measures carefully in your bench lab notebook, so should you record any analyses
+you do on the computer. With git, you "add" and "commit" changes that you make.
+These are analogous to writing down the values of your measurements and noting 
+how you took them. 
+
+The basic steps of the process are:
+
+1. **Fork** the [popgenInfo repository](https://github.com/NESCent/popgenInfo)\*
+2. **Clone** your fork to your local machine.\*
+6. Checkout your master branch, **fetch** the changes from NESCent and **merge**
+   them into your fork.
+3. Create a new **branch** and then **add** and **commit** new changes or content.
+4. **Push** your changes to your fork.
+5. Open a **pull request** from your fork to NESCent.
+
+\* These should only be done once.
+
+This particular tutorial will show you the basics of contributing to the
+popgenInfo website using the R package
+[*git2r*](https://cran.r-project.org/package=git2r), which was developed by the
+[rOpenSci project](https://ropensci.org/). This package allows for a nearly
+seamless and uniform interface for all contributors regardless of operating
+system.
+
+## Requirements
+
+We assume that you have already done a few things:
+
+1. Signed up for a [GitHub account](https://github.com)
+2. Install [R](https://cran.r-project.org)
+3. Install [Rstudio](https://rstudio.com/desktop/download)
+
+If you have also installed [git](https://git-scm.com/), you can set up Rstudio
+to work with git to help you commit changes, pull, and push. For the purposes of
+this tutorial, these things are not necessary.
+
+# Step 1: Forking the repository.
+
+Go to 
+[https://github.com/NESCent/popgenInfo](https://github.com/NESCent/popgenInfo) 
+and [click on the "Fork" 
+button](https://help.github.com/articles/fork-a-repo/#fork-an-example-repository).
+This will create a copy of the NESCent popgenInfo repository to your account.
+After you've forked the repository, you never have to fork it again!
+
+# Step 2: Clone your fork to your computer
+
+Just like forking, you only have to clone your fork to your computer once. It
+will live there until you decide to remove it.
+
+Now that you have your repository forked to your account, you will need to 
```

Comment:
This sentence reads awkward coming after the preceding one. Shouldn't it just come prior to the preceding paragraph rather than after it?